source/query/sort.txt

```diff
@@ -0,0 +1,88 @@
+.. _query-bar-sort:
+
+===========================
+Sort the Returned Documents
+===========================
+
+.. default-domain:: mongodb
+
+If the query bar displays the :guilabel:`Sort` option, you can specify
+the sort order of the returned documents.
+
+Set the Sort Order
+------------------
+
+To set the sort order:
+
+#. Click :guilabel:`Options`.
+
+#. Enter the ``sort`` document into the :guilabel:`Sort` field.
+
+   - To specify ascending order for a field, set the field to ``1`` in
+     the sort document.
+   - To specify descending order for a field, set
+     the field and ``-1`` in the sort documents.
+
+   .. example::
+
+      The following ``sort`` document sorts results first by ``year``
+      in descending order, and within each year, sort by ``name`` in
+      ascending order.
+
+      .. code-block:: javascript
+
+         { year: -1, name: 1 }
+
+   As you type, the :guilabel:`Find` button is disabled and the
+   :guilabel:`Sort` label turns red until a valid query is entered.
+
+   .. figure:: /images/querybar/query-sort-error.png
+      :figwidth: 300px
+      :lightbox:
+
+#.
+   .. include:: /includes/run-query.rst
```

Comment:
I think we should move the Note to be under the screenshot. It feels a bit awkward to have the step ("3"), then a semi-related note, then the screenshot associated with the step.

Reply:
Agreed. Fixed.